introduction_to_applying_machine_learning/visual_object_detection/visual_object_detection.ipynb

```diff
@@ -0,0 +1,2842 @@
+{
```

Comment:
<p>Can you make this section automatically clean-up?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I have changed to the following: </p><p> </p><p>If you run the notebook end-to-end, the Cleaning up section in the notebook will delete all the checkpoints and models automatically for you. If you choose to only train some of the four models in the notebook, please make sure to run corresponding code in the Cleaning up section to delete all the artifacts.</p><p> </p>